build/index.js

```diff
@@ -18,6 +19,18 @@ exports.onCreateProject = function (api, app, config, cb) {
     return fs.outputFileAsync(path.join(outpath, file), xmlDoc.toString(), 'utf-8');
   };
 
-  return copyCustomRules(config.outputPath)
-    .then(cb);
-}; 
+  if (config.target === 'native-android') {
+    return copyCustomRules(outputPath)
+      .then(cb);
+  } else if (config.target === 'native-ios') {
+    var buildScipt = path.join(__dirname, '../ios', 'buildScript'),
+      manifest = app.manifest.ios,
+      scriptContent = fs.readFileSync(buildScipt, 'utf-8');
+
+    outputPath = path.join(outputPath, 'xcodeproject');
```

Comment:
i think this path can be obtained from some property in app/config

Reply:
This is xcode project path which is fixed for devkit.It's been used in native-ios too.